tests/beaker_tests/lib/utilitylib.rb

```diff
@@ -268,7 +268,7 @@ def test_resource(tests, id, state=false)
 # Wrapper for config pattern-match tests
 def test_show_cmd(tests, id, state=false)
   stepinfo = format_stepinfo(tests, id, 'SHOW CMD')
-  show_cmd = get_vshell_cmd(tests[:show_cmd])
+  show_cmd = get_vshell_cmd(tests[id][:show_cmd])
```

Comment:
:+1: on your changes as is; just wondering if we can take this opportunity to get rid of `test_show_cmd` altogether at this point? Looks like only one test might actually still be using this and it's a aaa test...

`grep  test_show_cmd tests/* -r`
`grep grep show_pattern tests/* -r`